strings.po

```diff
@@ -8150,19 +8153,21 @@ msgstr "Umgebungsaustausch"
 #. STRINGS.DUPLICANTS.MODIFIERS.TERRIBLESLEEP.NAME
 msgctxt "STRINGS.DUPLICANTS.MODIFIERS.TERRIBLESLEEP.NAME"
 msgid "Dead Tired"
-msgstr ""
+msgstr "Tot müde"
 
 #. STRINGS.DUPLICANTS.MODIFIERS.TERRIBLESLEEP.TOOLTIP
 msgctxt "STRINGS.DUPLICANTS.MODIFIERS.TERRIBLESLEEP.TOOLTIP"
 msgid "This Duplicant didn't get any shuteye last night because {Disturber} made a racket"
-msgstr ""
+msgstr "Dieser Duplicant hat letzte Nacht kein Auge zugemacht, weil {Disturber} Krach gemacht hat"
 
 #. STRINGS.DUPLICANTS.MODIFIERS.TIME_REMAINING
 msgctxt "STRINGS.DUPLICANTS.MODIFIERS.TIME_REMAINING"
 msgid ""
 "\n"
 "Wears off in: {0}"
 msgstr ""
+"\n"
+"Nutz sich ab in: {0}"
```

Comment:
"Wear off" ist nicht wörtlich zu verstehen.
Eher sowas wie "Läuft ab in: " oder "Klingt ab in:"

Reply:
ändere ich in "Läuft ab in"